css/style.css

```diff
@@ -245,9 +245,8 @@ article{
   background-color: #fff;
   grid-column: 1;
   grid-row: 2;
-  padding: 5.5%;
+  padding: 110px 0 50px 50.5px; 
```

Comment:
Don't use half pixels! Browsers sometimes try to simulate what you meant, but pixels can only be whole. So use only whole numbers.